packages/zowe-explorer/src/utils/AuthUtils.ts

```diff
@@ -11,84 +11,99 @@
 
 import * as util from "util";
 import * as vscode from "vscode";
-import { imperative, Gui, MainframeInteraction, IZoweTreeNode } from "@zowe/zowe-explorer-api";
+import { imperative, Gui, MainframeInteraction, IZoweTreeNode, ErrorCorrelator, ZoweExplorerApiType, CorrelatedError } from "@zowe/zowe-explorer-api";
 import { Constants } from "../configuration/Constants";
 import { ZoweLogger } from "../tools/ZoweLogger";
 import { SharedTreeProviders } from "../trees/shared/SharedTreeProviders";
 
+export interface ErrorContext {
```

Comment:
not sure we need to export it since we don't seems to use it anywhere else in the ZE package 😋

Reply:
ah, good point, thanks - will address in commit with other feedback addressed 🙂